metrics/core.go

```diff
@@ -117,3 +94,56 @@ func CreateMetrics() *Metrics {
 
 	return m
 }
+
+func CreatePrometheusMetrics(cfg config.PrometheusMetrics) *PrometheusMetrics {
+	cacheWriteTimeBuckts := []float64{0.001, 0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5, 1}
+	registry := prometheus.NewRegistry()
+	promMetrics := &PrometheusMetrics{
+		Registry: registry,
+		RequestDurationMetrics: newHistogramVector(cfg, registry,
```

Comment:
If `method` here is PUT vs GET, I would recommend to separate these out into their own metrics. Read and write operations are inherently separate and I don't see value in sharing the same metric name. I also think we want to separate front end (calls into Prebid Cache) vs backend requests (calls out to the data store). 

What are the possible values for the `result` label? We need to be careful since each one will use 13 counters (10 for the buckets, 1 for the infinity bucket, 2 for the counter and sum counters).

Reply:
corrected